docs/_includes/expressive-loader.html

```diff
@@ -0,0 +1,632 @@
+          <div class="expressive-line-animation__line expressive-line-animation__line--blue" style="width: 28%;"></div>
+        </div>
+      </div>
+    </div>
+    </div>
+  </div>
+
+  {% highlight html %}
+  <div class="expressive-loader expressive-loader--medium">
+    <div class="expressive-loader__message-container">
+      <div aria-hidden="true" class="expressive-loader__message expressive-loader__message--next">
+        Just a moment longer.
+      </div>
+      <div role="status" id="c3-message" class="expressive-loader__message">
+        We're finishing your order.
+      </div>
+    </div>
+    <div role="progressbar" aria-label="Loading..." aria-describedby="c3-message" class="expressive-line-animation">
+      <div class="expressive-line-animation__standard">
+        <div
+          class="expressive-line-animation__line expressive-line-animation__line--blue expressive-line-animation__line--animated"
+          style="width: 14%;"></div>
+        <div
+          class="expressive-line-animation__line expressive-line-animation__line--yellow expressive-line-animation__line--animated"
+          style="width: 19%;"></div>
+        <div
+          class="expressive-line-animation__line expressive-line-animation__line--green expressive-line-animation__line--animated"
+          style="width: 37%;"></div>
+        <div
+          class="expressive-line-animation__line expressive-line-animation__line--red expressive-line-animation__line--animated"
+          style="width: 26%;"></div>
+        <div
+          class="expressive-line-animation__line expressive-line-animation__line--blue expressive-line-animation__line--animated"
+          style="width: 24%;"></div>
+        <div
+          class="expressive-line-animation__line expressive-line-animation__line--yellow expressive-line-animation__line--animated"
+          style="width: 34%;"></div>
+        <div
+          class="expressive-line-animation__line expressive-line-animation__line--green expressive-line-animation__line--animated"
+          style="width: 13%;"></div>
+        <div class="expressive-line-animation__line expressive-line-animation__line--red" style="width: 15%;"></div>
+        <div class="expressive-line-animation__line expressive-line-animation__line--blue" style="width: 28%;"></div>
+      </div>
+    </div>
+  </div>
+  {% endhighlight %}
+
+
+  <h3 id="expressive-loader-reduced-motion-base">Base Expressive Loader with Reduced Motion</h3>
+  <p>
+    When the user prefers reduced motion, the lines still animate. Instead of growing from 0% to a random size,
+    they stay a fixed size and scroll slowly across the screen. No Javascript is needed to achieve this effect.
+  </p>
+
+  <div class="demo">
+    <div class="demo__inner">
+      <div class="expressive-loader">
+        <div role="progressbar" aria-label="Loading..." class="expressive-line-animation">
+          <div class="expressive-line-animation__reduced-motion">
```

Comment:
It's probably not ideal to have different markup for reduced motion since it's a client site setting that is queried in `css`. Is it possible to come up with a singular markup structure that would work in both cases - regular and reduced motion?

Reply:
Agreed. I think this is the main change that is required right now. Getting it all onto a single, flexible markup structure (even if it might end up containing a little bit of redundancy).